site/security_users_groups_roles.md

```diff
@@ -0,0 +1,53 @@
+### Modeling WebLogic Users, Groups and Roles
+WebLogic Server has the ability to establish a set of Users, Groups and Global Roles as part of the WebLogic domain creation. The WebLogic global roles become part of the WebLogic role mapper (i.e. `XACMLRoleMapper`) and are specified under `domainInfo` in the `WLSRoles` section. The users and groups become part of the Embedded LDAP server (i.e. `DefaultAuthenticator`) and are specified under `topology` in the `Security` section.
```

Comment:
Users, Groups and Global Roles -> users, groups, and global roles (these terms do not need to be capitalized)